projects/1.19/assets/tanukidecor/tanukidecor/lang/zh_cn.json

```
@@ -0,0 +1,245 @@
+{
+  "itemGroup.tanukidecor": "狸猫装饰",
+
+  "block.tanukidecor.alarm_clock": "闹钟",
+  "block.tanukidecor.anniversary_clock": "周年纪念钟",
+  "block.tanukidecor.antique_clock": "古董时钟",
+  "block.tanukidecor.banjo_clock": "班卓琴钟",
+  "block.tanukidecor.blue_clock": "蓝色时钟",
+  "block.tanukidecor.carriage_clock": "马车时钟",
+  "block.tanukidecor.crystal_clock": "石英钟",
+  "block.tanukidecor.cuckoo_clock": "布谷鸟钟",
+  "block.tanukidecor.display_watch": "展示钟",
+  "block.tanukidecor.emblem_clock": "徽章钟",
+  "block.tanukidecor.foliot_clock": "摆锤钟",
+  "block.tanukidecor.gingerbread_clock": "姜饼钟",

```

```suggestion
  "block.tanukidecor.gingerbread_clock": "姜饼时钟",
```
考虑到是座钟才这么翻，改不改随你
要改的话就查一遍全文吧